responsive layout, first page

VictorRadoi opened the pull request:
responsive layout added and some parts from first page

VictorRadoi committed:
responsive layout, first page

responsive layout added and some parts from first page

VictorRadoi commented on website/lib/presentation/desktop/home_page/home_page_desktop.dart:
fnf

VictorRadoi committed:
routes, home_page done, rest of the pages

routes added for all the pages
home page and pop are done
other pages created

VictorRadoi committed:
normal cv page done

normal and boring cv done

VictorRadoi committed:
terminal page done

VictorRadoi committed:
minor changes

image added over terminal
padding added to terminal

LucaDorinAnton commented on new_flutter_website/.fvmrc:
```diff
@@ -0,0 +1,3 @@
+{
```
Can we call this folder `flutter`?

LucaDorinAnton commented on new_flutter_website/.gitignore:
include to root `.gitignore`, 
also please add to it:
- .vscode
- android
- ios
- linux
- macos
- windows 
- website
- web

also please remove from a PR (I can't review a PR with 300 files in it)

(I tried running the build without those and it worked)

LucaDorinAnton reviewed (changes requested):
Please clean up PR so that I can review it thoroughly

VictorRadoi committed:
files deleted

macos, linux, android, windows files deleted

VictorRadoi committed:
new home page design

a new app bar introduced for home, terminal and cv pages
a new home page design

VictorRadoi committed:
new pages

about and contact me pages introduced
new button on home screen for CVs
new buttons in app bars

VictorRadoi committed:
Update contact_me_desktop.dart
